scene/gui/tab_container.cpp

```diff
@@ -479,7 +481,14 @@ void TabContainer::set_current_tab(int p_current) {
 	}
 
 	_change_notify("current_tab");
-	emit_signal("tab_changed",current);
+
+	if (pending_previous == current)
+		emit_signal("tab_selected", current);
```

Comment:
That's what was suggested in the issue #7752 , I think it is better than tab_clicked, because if it was used in an environment that doesn't have mouse (i.e, touchscreen) then that wouldn't make sense.

Besides, current behavior didn't change with these changes. So it is safe and solves the issue while retaining current behavior which I need.

Also, since I long deleted the fork and don't own it, how am I supposed to resolve the conflict?